src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const figureOne = Big(numberOne);
+  const figureTwo = Big(numberTwo);
+  if (operation === '+') {
+    return figureOne.plus(figureTwo).toString();
+  }
+  if (operation === '-') {
+    return figureOne.minus(figureTwo).toString();
+  }
+  if (operation === 'x') {
+    return figureOne.times(figureTwo).toString();
+  }
+  if (operation === '÷') {
+    return figureOne.div(figureTwo).toString();
+  }
+  if (operation === '%') {
+    return figureOne.mod(figureTwo).toString();
```

Comment:
* [ ]  `.mod` is not the right conversion formula for real numbers to decimal numbers. Let us multiply the figureOne by `0.01` instead.
> Let us keep in mind that when the user selects `%` when only `figureOne` is present at the time of selection it shoul return the percentage value of figure one, if both `figureOne` and figureTwo Exist at the time of selection then, return the result of the operation in decimal numerals.

Reply:
this is not a percentage operator. this is the modulo operator.